.ci_support/linux_64_.yaml

```diff
@@ -5,4 +5,4 @@ channel_targets:
 docker_image:
 - quay.io/condaforge/linux-anvil-x86_64:alma9
 python_min:
-- '3.11'
```

Comment:
Hmm conda-forge bot _really_ doesn't want to leave this alone.... Might need to hard code it in the recipe instead?